src/test/java/spoon/test/refactoring/RefactoringTest.java

```diff
@@ -89,4 +90,25 @@ public boolean matches(CtInvocation<?> element) {
 		}).get(0);
 		assertEquals("this(\"\")", thisInvocation.toString());
 	}
+
+
+        @Test
+        public void testTransformedInstanceofAfterATransformation() throws Exception {
+                final Launcher launcher = new Launcher();
+                launcher.setArgs(new String[] {
+                                "-i", "src/test/java/spoon/test/refactoring/testclasses",
+                                "-o", "target/spooned/refactoring",
+                                "-p", ThisTransformationProcessor.class.getName()
+                });
+                launcher.run();
+                final CtClass<?> aClassX = (CtClass<?>) launcher.getFactory().Type().get("spoon.test.refactoring.testclasses.AClassX");
+
+                final CtBinaryOperator<?> instanceofInvocation = aClassX.getElements(new AbstractFilter<CtBinaryOperator<?>>(CtBinaryOperator.class) {
```

Comment:
There is a better way to do that: use `new TypeFilter<CtBinaryOperator<?>>(CtBinaryOperator.class)`.